app/views/shared/check_answers/_linking_and_copying.html.erb

```diff
@@ -1,73 +1,55 @@
           row.with_value { safe_yes_or_no(@legal_aid_application&.lead_linked_application&.confirm_link? || t("generic.no")) }
-          unless read_only
-            row.with_action(
-              text: t("generic.change"),
-              href: providers_legal_aid_application_link_application_make_link_path(@legal_aid_application),
-              visually_hidden_text: t(".section_linking.linked"),
-            )
-          end
+          row.with_action(
+            text: t("generic.change"),
+            href: providers_legal_aid_application_link_application_make_link_path(@legal_aid_application),
+            visually_hidden_text: t(".section_linking.linked"),
+          )
         end
         if @legal_aid_application&.lead_linked_application&.confirm_link?
           summary_list.with_row(classes: "app-check-your-answers__linking_case") do |row|
             row.with_key(text: t(".section_linking.#{@legal_aid_application.lead_linked_application.link_type_code}"))
             row.with_value(text: sanitize(@legal_aid_application.lead_linked_application.target_application.link_description))
-            unless read_only
-              row.with_action(
-                text: t("generic.change"),
-                href: providers_legal_aid_application_link_application_find_link_application_path(@legal_aid_application),
-                visually_hidden_text: [t(".section_linking.aria_prefix"), t(".section_linking.#{@legal_aid_application.lead_linked_application.link_type_code}")].join(" "),
-              )
-            end
+            row.with_action(
+              text: t("generic.change"),
+              href: providers_legal_aid_application_link_application_find_link_application_path(@legal_aid_application),
+              visually_hidden_text: [t(".section_linking.aria_prefix"), t(".section_linking.#{@legal_aid_application.lead_linked_application.link_type_code}")].join(" "),
+            )
           end
         end
-      end %>
-  <% if @legal_aid_application&.lead_linked_application&.confirm_link? && all_linked_applications_details(@legal_aid_application).present? %>
-    <div class="govuk-grid-row" id="app-check-your-answers__other_links">
-      <div class="govuk-grid-column-full">
-        <h2 class="govuk-heading-m"><%= t ".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}" %></h2>
-      </div>
-    </div>
-    <%= govuk_summary_list(card: { title: t(".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}") }, actions: false) do |summary_list|
+      end
+    end %>
+
+<% if @legal_aid_application&.lead_linked_application&.confirm_link? && all_linked_applications_details(@legal_aid_application).present? %>
+  <%= govuk_summary_card(title: t(".other_links_section.#{@legal_aid_application.lead_linked_application.link_type_code}")) do |card|
+        card.with_summary_list(actions: false) do |summary_list|
```

Comment:
This is `actions: false`, but line 3 has `actions: !read_only` should they be the same or do we never want a change link for this?

Reply:
This section shows the following
```
other_links_section:
  FC_LEAD: All applications with a family link to this one
  LEGAL: All applications with a legal link to this one
````

As it's showing other applications linked to this, I would expect it to never have a change link